pkg/kubelet/dockertools/manager.go

```diff
@@ -502,11 +505,13 @@ func (dm *DockerManager) GetPodStatus(pod *api.Pod) (*api.PodStatus, error) {
 		_, err := dm.client.InspectImage(image)
 		if err == nil {
 			containerStatus.State.Waiting = &api.ContainerStateWaiting{
-				Reason: fmt.Sprintf("Image: %s is ready, container is creating", image),
+				Reason:  "ContainerCreating",
+				Message: fmt.Sprintf("Image: %s is ready, container is creating", image),
 			}
 		} else if err == docker.ErrNoSuchImage {
 			containerStatus.State.Waiting = &api.ContainerStateWaiting{
-				Reason: fmt.Sprintf("Image: %s is not ready on the node", image),
+				Reason:  "NoSuchImage",
```

Comment:
@bgrant0607 @dchen1107  I'll look into the reason cache and make another pr to make reason and message keep consistant with this.